cmake/common.cmake

```diff
@@ -124,9 +124,16 @@ function(configure_clang_and_clang_tidy_settings _TARGET)
 
   if(CLANG_TIDY_AVAILABLE)
```

Comment:
nothing in the commit log to describe why these changes are needed, suggest pulling out into a separate PR along with the issue that these changes are addressing.

Reply:
(not sure how to comment on a specific file, the above comment refers to `cmake/common.cmake` and all the changes in there.)

Reply:
Agreed with your first comment. It comes from https://github.com/cellml/libcellml/pull/761/files, which I need on my Mac since I am now on LLVM 11. So, if anything, I should have made it clear that this PR relies on PR #761.